lib/gql/display-name.ts

```diff
@@ -0,0 +1,22 @@
+import { FullContext, Sdk } from "@zeitgeistpm/sdk-next";
+
+export const getDisplayName = async (
+  sdk: Sdk<FullContext>,
+  addresses: string[],
+) => {
+  const identities = await Promise.all(
+    addresses.map((address) => sdk.api.query.identity.identityOf(address)),
+  );
+
+  const textDecoder = new TextDecoder();
+
+  const names: (string | null)[] = identities.map((identity) =>
+    identity.isNone === false
+      ? textDecoder.decode(
+          (identity.value.get("info") as any).get("display").value,
+        )
+      : null,
+  );
```

Comment:
I think maybe this would be a bit more performant and more idiomatic to polkadotjs?
```suggestion
  const identities = await sdk.api.query.identity.identityOf.multi(addresses);
  const names: (string | null)[] = identities.map(
    (identity) => identity.unwrapOr(null)?.info?.display?.toString() ?? null,
  );
```

Reply:
I think I did something like this before but it doesn't handle all characters correctly like Tvrtko's full name